source/_integrations/withings.markdown

```diff
@@ -40,8 +40,9 @@ Once saved, the "Client Id" and "Consumer Secret" fields will be populated. You
     withings:
       client_id: CLIENT_ID
       client_secret: CONSUMER_SECRET
+      use_webhook: true
```

Comment:
```suggestion
```
In the first example, we would like to see only config variables that are required.